utilities/constants.py

```diff
@@ -41,18 +41,18 @@
     "Sidearm":                  "Big 10 Athletics",
     "Campus Visits":            "Campus Visits",
     "Professional Development": "Career Development",
-    "Ceremonies & Services":    "Ceremonies & Services",
+    "Ceremonies & Services":    "Ceremonies and Services",
```

Comment:
```suggestion
    "Ceremonies & Services":    "Ceremonies and Services",
    "Ceremonies and Services":    "Ceremonies and Services",
```

Reply:
@minump May I know why this suggested change?

Reply:
coz the mapping has changed. EventsBB has "Ceremonies and Services" instead of "Ceremonies & Services". So new events would only have "and ". I also think the mapping `"Ceremonies & Services" : "Ceremonies and Services" should remain for backward compatability.

Reply:
`"Ceremonies and Services": "Ceremonies and Services"` looks right. I don't think `"Ceremonies & Services": "Ceremonies and Services"` is needed as we have not put Events BB with category name changes on PROD before.

Reply:
@bingzhang, Could you please update the branch based on the above comment? I think this PR can be merged after that. Thanks.